src/lib.rs

```diff
@@ -284,20 +284,20 @@
     doc = "[`calendar`]: widgets::calendar::Monthly"
 )]
 //!
-//! [Ratatui Book]: https://ratatui.rs
-//! [Installation]: https://ratatui.rs/installation.html
-//! [Rendering]: https://ratatui.rs/concepts/rendering/index.html
-//! [Application Patterns]: https://ratatui.rs/concepts/application_patterns/index.html
-//! [Hello World tutorial]: https://ratatui.rs/tutorial/hello_world.html
-//! [Backends]: https://ratatui.rs/concepts/backends/index.html
-//! [Widgets]: https://ratatui.rs/how-to/widgets/index.html
-//! [Handling Events]: https://ratatui.rs/concepts/event_handling.html
-//! [Layout]: https://ratatui.rs/concepts/layout/index.html
-//! [Styling Text]: https://ratatui.rs/how-to/render/style-text.html
-//! [rust-tui-template]: https://github.com/ratatui-org/rust-tui-template
-//! [ratatui-async-template]: https://ratatui-org.github.io/ratatui-async-template/
+//! [Ratatui Book]: https://ratatui.rs/introduction
+//! [Installation]: https://ratatui.rs/installation
+//! [Rendering]: https://ratatui.rs/concepts/rendering
+//! [Application Patterns]: https://ratatui.rs/concepts/application-patterns
+//! [Hello World tutorial]: https://ratatui.rs/tutorials/hello-world
+//! [Backends]: https://ratatui.rs/concepts/backends
+//! [Widgets]: https://ratatui.rs/how-to/widgets
+//! [Handling Events]: https://ratatui.rs/concepts/event-handling
+//! [Layout]: https://ratatui.rs/how-to/layout
+//! [Styling Text]: https://ratatui.rs/how-to/render/style-text
+//! [ratatui-template]: https://github.com/ratatui-org/ratatui-template
+//! [ratatui-async-template]: https://ratatui-org.github.io/ratatui-async-template
```

Comment:
The website canonicalizes urls with a final slash, so it wouldn't hurt to add these here...

```suggestion
//! [Ratatui Book]: https://ratatui.rs/introduction/
//! [Installation]: https://ratatui.rs/installation/
//! [Rendering]: https://ratatui.rs/concepts/rendering/
//! [Application Patterns]: https://ratatui.rs/concepts/application-patterns/
//! [Hello World tutorial]: https://ratatui.rs/tutorials/hello-world/
//! [Backends]: https://ratatui.rs/concepts/backends/
//! [Widgets]: https://ratatui.rs/how-to/widgets/
//! [Handling Events]: https://ratatui.rs/concepts/event-handling/
//! [Layout]: https://ratatui.rs/how-to/layout/
//! [Styling Text]: https://ratatui.rs/how-to/render/style-text/
//! [ratatui-template]: https://github.com/ratatui-org/ratatui-template
//! [ratatui-async-template]: https://ratatui-org.github.io/ratatui-async-template
```